src/components/common/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -34,19 +34,52 @@
         </ul>
       </article>
 
-      <!-- Proprietor or partners section -->
-      <article v-if="isChangeFiling" class="section-container">
+      <!-- Change or conversion section -->
+      <article v-if="isChangeFiling || isConversionFiling" class="section-container">
         <span class="info-text">{{ orgPersonSubtitle }}</span>
 
-        <!-- Sole Prop Help -->
         <HelpSection
-          v-if="getResource.entityType === CorpTypeCd.SOLE_PROP"
+          v-if="orgPersonHelp"
```

Comment:
This is not specific to SP. Now, display this component if the help object is not null.